src/Factories/EnvironmentSetup/ProductionEnvironmentSetup.php

```diff
@@ -9,23 +9,30 @@
 use Monolog\Handler\StreamHandler;
 use Monolog\Logger;
 use Psr\Log\LoggerInterface;
+use Symfony\Component\Cache\Adapter\FilesystemAdapter;
+use Symfony\Component\Cache\Psr16Cache;
 use WMDE\Fundraising\Frontend\Factories\FunFunFactory;
 
 class ProductionEnvironmentSetup implements EnvironmentSetup {
 
 	public function setEnvironmentDependentInstances( FunFunFactory $factory ) {
 		$this->initializeLoggers( $factory );
-
-		$factory->enableCaching();
 	}
 
 	private function initializeLoggers( FunFunFactory $factory ) {
+		$this->setCampaignCache( $factory );
```

Comment:
This is not a logger, please move the method call to `setEnvironmentDependentInstances`